README.md

```diff
@@ -39,6 +39,11 @@ Then it can be used passing in numeric values to the `toWords()` method:
 ```php
 $numberTransformer->toWords(5120); // outputs "five thousand one hundred twenty"
 ```
+You can also use the fluent version:
+
+```php
+NumberToWords::numberTransformer('en')->toWords(5120) // outputs "five thousand one hundred twenty"

```

Comment:
I think this requires an update according to the latest changes (static methods).